### DISCUSSION TITLE
[midi-visualizer] piano playing without the roll

### DISCUSSION THREAD

**Discussion by btopro** [cifkao/html-midi-player#3] [Q&A] [answered]
I see that `waterfall` has the piano playing and that `piano-roll` has the sheet music rolling forward; is there any option to just get the piano (like in waterfall) but without the roll?

Awesome element, going to be digging in more today. We make editors for educators and I work in an arts college and they've asked for something like this for some time

**Comment by cifkao** [comment, marked as answer]
You could just hide the waterfall like
```css
.waterfall-notes-container {
    display: none;
}
```

Otherwise it's more of a [@magenta/music](https://github.com/magenta/magenta-js/) issue. If they implement a visualizer which is just a piano keyboard, it will be easy to add it here.

**Comment by btopro** [comment]
oh awesome thank you I'll add a setting to my element to support that idea
